README.md

```diff
@@ -1,10 +1,102 @@
+### Running locally
+
+TODO How to get this project up and running locally.
+
+## Project Scope
+From a UX perspective, the idea solution is to get back data as fast as possible when someone adds a URL. So it would spit back the simple data first (url, image, description, medium type), and if it's new and needs to be enriched, we enrich it in the background by hitting the appropriated APIs.
+
+So maybe there are two endpoints, one with a quick response (no enriching) and one that does the full enrichment. We can discuss and figure out the best solution together.
+
+Supports the following URLs and APIs. Example URLs linked.
+
+| Medium | URLs | APIs |
+| :-- | :-- | :-- |
+| Book | https://goodreads.com <br> https://amazon.com | [GoodReads API](https://www.goodreads.com/api) for data, authors, topics<br>[OpenLibrary](https://openlibrary.org/developers/api) for image covers<br>Amazon solution TBD |
+| Music<br>Podcast | Spotify ([song](https://open.spotify.com/track/7sd72KZS8D59g5NmhxyHpJ), [album](https://open.spotify.com/album/2xkZV2Hl1Omi8rk2D7t5lN)) <br>Apple ([show](https://podcasts.apple.com/us/podcast/the-tim-ferriss-show/id863897795), [episode](https://podcasts.apple.com/us/podcast/429-nick-kokonas-on-resurrecting-restaurants-skin-in/id863897795)) | [Spotify API](https://developer.spotify.com/documentation/web-api/)<br>Apple TBD |
+| Film<br>TV | IMDB ([film](https://www.imdb.com/title/tt0051808/), [show](https://www.imdb.com/title/tt0475784/), [episode](https://www.imdb.com/title/tt4227538/)) | OMDB API for data<br>[TMDB API](https://www.themoviedb.org/documentation/api) for people, trailers, etc |
+| Art | [Artsy](https://www.artsy.net/artwork/evan-nesbit-lemon-drift) | [Artsy API](https://developers.artsy.net/v2/) |
+
+TODO (Joel will write this) add ideal JSON response object
+
+## Project Milestones
+
+### V1 Setup
+- [x] Setup initial code base
+- [x] Migrate to Analogue github org
+- [x] Set up initial APIs
+- [x] Deploy on Heroku
+- [ ] Integrate [Sentry addon](https://elements.heroku.com/addons/sentry) for production errors
+- [ ] Add todos and scope to README
+- [ ] Add local setup to README
+
+### V2 Optimize
+- [ ] Research if existing API wrappers exist in Python ([for example](https://github.com/mdzhang/goodreads-api-client-python))
+- [ ] Potentially turn APIs in objects
+- [ ] Endpoint for a quick response
+- [ ] Build Analogue specific JSON response object
+- [ ] Using Streaming API/Fanner model to optimize speed and handle jobs in background
+- [ ] Error handling
+- [ ] URL normalization (should correctly handle/remove query params in the url string)
+- [ ] Handle other URLs from same URL domain
+- [ ] Secure solution for handling API keys using [dotenv](https://github.com/theskumar/python-dotenv)
+
+### V3 Support more sources
+- [ ] Amazon
+- [ ] Podcast and Music from Apple Music
+- [ ] Artsy APIs
+- [ ] Soundcloud API
+
+### V4 Connnect
+TODO. We can plan through the integration with Rails app soon.
+
+## Commit Guide
```

Comment:
@FaazAbidi @alihaiderrizvi  FYI, I added an emoji commit guide here. I think it's a good idea to adopt clean commit practices since it really helps with quickly scanning git history. It also helps future developers. Especially as you start to work on many different projects, it really helps to leave these breadcrumbs for your future self and other developers.

I personally like emojis since they're visual and easy to scan, and once you get the mapping down it's easy to see what type of work has been done. Another common paradigm here instead of emojis are to use short descriptors like `feat: Added a new feature` or `bug: fixed bug with data scraper` etc. 

I know process like this can sometimes slow you down in the short run, but in the long run it's extremely helpful in my opinion. Especially when projects start having 100+ commits. Your future self will thank you.

Plus it just looks so pretty :) 

<img width="380" alt="Screen Shot 2020-05-06 at 1 41 20 PM" src="https://user-images.githubusercontent.com/217719/81226352-90cac000-8f9f-11ea-8097-07d61c50a572.png">

cc @hxghhhh